fastlane/Fastfile

```diff
@@ -233,6 +234,37 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       dsym_path: "./WooCommerce.app.dSYM.zip",
     )
     sh("cd .. && rm WooCommerce.app.dSYM.zip")
+
+    if (options[:create_release])
+      archive_zip_path = File.dirname(Dir.pwd) + "/WooCommerce.xarchive.zip"
+      zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
+
+      version = ios_get_app_version()
+      create_release(repository:GHHELPER_REPO, 
+        version: version, 
+        release_notes_file_path:'../WordPress/Resources/release_notes.txt',
```

Comment:
Oh – also, I think this doesn't need the `../` part since it's not in the `Scripts` directory like WP?